rust_src/src/alloc.rs

```diff
@@ -112,4 +114,20 @@ pub fn purecopy(obj: LispObject) -> LispObject {
     }
 }
 
+/// Return a newly allocated marker which does not point to any place.
+#[lisp_fn(c_name = "make_marker", name = "make-marker")]
+pub fn make_marker_lisp() -> LispObject {
```

Comment:
You can simply name the function `make_marker` and remove the `c_name` and `name` from the `lisp_fn`, those are only in case the desired name is already taken.